### ISSUE TITLE
Product Details tab is missing

### ISSUE THREAD

**Issue by addison74** [magento/magento2#10742]
I am picking up as example a product with the attribute set named Top. Inside this attribute set you can see the first tab is called [Product Details].

![attribute_set](https://user-images.githubusercontent.com/8360474/29989362-237e73e2-8f7b-11e7-8eb7-3a7035997995.jpg)

Now go to edit a product with Top attribute set assigned. The [Product Details] tab is missing. This creates an annoying situation in case [Product Details] has too many options like in the sample pack. I prefer to close this tab (if will exist) to focus on the other tabs better. It is not normal spending time scrolling for tabs because there is no hiding option for [Product Content] tab. Comparing with Magento 1 this is a layout issue:

![product_content](https://user-images.githubusercontent.com/8360474/29989401-55eec534-8f7b-11e7-987b-47a1f0400be8.jpg)

By default [Product Details] tab should be open, all the other tabs should be closed. This is not happening with [Configurations] tab. It is open:

![configuration_tab](https://user-images.githubusercontent.com/8360474/29989425-76b88ade-8f7b-11e7-8f25-a0691deb46d6.jpg)

**Preconditions**: Magento Version 2.1.8
**Steps to reproduce**: Go visiting Products and Categories sections in backend
**Expected result**: In that attribute set there is a group called Product Content. It is not appearing as the others groups, with name and having the option to toggle it on/off. This missing feature creates an "always visible" area for product information. 
**Actual results**: You cannot hide the first attribute group. This is becoming annoying because you have to scroll to much every time, then you have always visible information about the product which distract. In Magento 1 every section was in a tab, General tab was not always visibile

Same situation for Categories, I cannot hide the first section, it stays always "open"

**Comment by addison74** [comment]
The same situation I found for Categories section too. Those 3 options (Enable Category, Include in Menu, Category Name) are not in a tab. 

For product editing page I prefer the tabs like in [Customer Information] section, on the left side. You have to adopt the same layout all over the Backend, not different layouts. How I enter the data in [Customer Information] section is different than [Product Edit] section.

**Comment by magento-engcom-team** [comment]
@ADDISON74 thank you for your report.
Please use the issue [reporting guidelines](https://github.com/magento/magento2/wiki/Issue-reporting-guidelines) to report an issue. Please, edit issue description and include the preconditions, the steps required to reproduce, the actual result, and the expected result.
Please, add the following information:
 - [x] Preconditions: Magento version
 - [x] Steps to reproduce
 - [x] Expected result
 - [x] Actual result

**Comment by addison74** [comment]
**Magento Version**: 2.1.8
**Steps to reproduce**: Go visiting Products and Categories sections in backend
**Expected result**: In that attribute set there is a group called Product Content. It is not appearing as the others groups, with name and having the option to toggle it on/off. This missing feature creates an "always visible" area for product information. 
**Actual results**: You cannot hide the first attribute group. This is becoming annoying because you have to scroll to much every time, then you have always visible information about the product which distract. In Magento 1 every section was in a tab, General tab was not always visibile

Same situation for Categories, I cannot hide the first section, it stays always "open"

**Comment by addison74** [comment]
More information you can find here:

https://magento.stackexchange.com/questions/192099/am-i-the-only-one-not-happy-with-magento-2-administrators-interface

We need the feature in order to update the information about Products and Categories very fast, without scrolling, keeping open only the information we need by hiding a group or a section. After 8 years of Magento 1, this thing came in front immediately, no option to hide the first section in Product and Category page.

**Comment by magento-engcom-team** [comment]
@ADDISON74, thank you for your report.
This seems to be correct Magento behavior. Please refer to the [Community Forums](https://community.magento.com/) or the [Magento Stack Exchange](https://magento.stackexchange.com/) site for advice or general discussion about this.
Otherwise you may submit Pull Request with the suggested changes.

**Comment by magento-engcom-team** [comment]
@ADDISON74 Unfortunately we are closing this ticket now, as github issue tracker is only intended for Magento 2 core technical issues. Please refer to the [Community Forums](https://community.magento.com/) to submit a feature request.

Thank you for collaboration.

**Comment by ADDISON74** [comment]
Comparing with Magento 1 when editing a product in Backend Magento 2 takes too much scrolling on a page and the UX is still a basic one. The layout of the page is still not responsive. 

Without a collapsible option and its name for the first tab I managed to let only tow options there: Product Name and Enable Product. In this way all the other attributes were going in a dedicated tab and the window space is optimized. 

I came back to this post after 3 years after I installed Magento 2.4.2. Pretty sad to see the are no improvements in Backend related to UX. Even the categories icons are from Magento 1 and a change was requested 5 years ago.
